docs/content/docs/dev/table/concepts/timezone.md

```diff
@@ -0,0 +1,311 @@
+```
+
+```
++----+-------------------------+-------------------------+-------------------------+-------------------------+
+| op |                       c |                  EXPR$1 |               EXPR$2    |                  EXPR$3 |
++----+-------------------------+-------------------------+-------------------------+-------------------------+
+| +I | 1970-01-01 08:00:04.001 | 1970-01-01 08:00:04.001 | 1970-01-01 08:00:04.001 | 1970-01-01 00:00:01.001 |
++----+-------------------------+-------------------------+-------------------------+-------------------------+
+```
+
+## Time Attribute and Time Zone
+
+Please see [Time Attribute]({{< ref "docs/dev/table/concepts/time_attributes">}}#date-and-time) for more information about time attribute.
+
+### Processing Time Attribute and Time Zone
+Flink SQL definine process time attribute by function PROCTIME(), the function return type is TIMESTAMP_LTZ.
+
+**NOTE**
+Before Flink 1.13, the function return type of PROCTIME() is TIMESTAMP, and the return value is the TIMESTAMP in UTC time zone,
+E.g. the PROCTIME() returns `2021-03-01 04:00:00` when user is in Shanghai and current timestamp is `2021-03-01 12:00:00 (UTC+8)`. 
+Flin 1.13 fixes this issue and uses TIMESTAMP_LTZ type as return type of PROCTIME(), users don't need to deal time zone proctime anymore.
+
+The PROCTIME() always represents your local timestamp value, using TIMESTAMP_LTZ type can also support DayLight Saving Time well. 
+
+```sql
+Flink SQL> SET table.local-time-zone=UTC; 
+Flink SQL> SELECT PROCTIME();
+```
+```
++----+-------------------------+
+| op |                  EXPR$0 |
++----+-------------------------+
+| +I | 2021-04-15 14:48:31.387 |
++----+-------------------------+
+```
+
+```sql
+Flink SQL> SET table.local-time-zone=UTC+8; 
+Flink SQL> SELECT PROCTIME();
+```
+```
++----+-------------------------+
+| op |                  EXPR$0 |
++----+-------------------------+
+| +I | 2021-04-15 22:48:34.567 |
++----+-------------------------+
+```
+```sql
+Flink SQL> CREATE TABLE src (
+             a INT,
+             b DOUBLE,
+             proctime as PROCTIME()
+           ) WITH (
+             'connector' = 'kafka',
+             ...);
+Flink SQL>  SELECT 
+                TUMBLE_START(proctime, INTERVAL '0.003' SECOND),
+                TUMBLE_END(proctime, INTERVAL '0.003' SECOND),
+                TUMBLE_PROCTIME(proctime, INTERVAL '0.003' SECOND),
+                COUNT(a),
+                SUM(b)
+            FROM src
+            GROUP BY TUMBLE(proctime, INTERVAL '0.003' SECOND);
+ -- the TUMBLE_START    column will be close to 2021-04-15 22:48:34.567, the type of TUMBLE_START    column is TIMESTAMP
+ -- the TUMBLE_END      column will be close to 2021-04-15 22:48:34.570, the type of TUMBLE_END      column is TIMESTAMP      
+ -- the TUMBLE_PROCTIME column will be close to 2021-04-15 22:48:34.570, the type of TUMBLE_PROCTIME column is TIMESTAMP_LTZ      
```

Comment:
This is hard to understand. Would be better to show some result example here and explain return type of `TUMBLE_START`, `TUMBLE_END`, `TUMBLE_PROCTIME` separately.